text/0032-nft-standard.md

```diff
@@ -93,13 +122,66 @@ The AttributeObject has the following JSON structure:
 
 The AttributeObject may have more attributes than the ones described above but those are the required ones.
 
+- The RoyaltyObject has the following JSON structure:
+
+```
```

Comment:
To keep the same standard as in the `AttributeObject`, we must have `{` to start the object and `}` to close it. Similar to what you did in the `CreatorObject`.